gitlab.go

```diff
@@ -497,6 +497,34 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 	return response, err
 }
 
+// Lint validates .gitlab-ci content
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/lint.html
+func (c *Client) Lint(content string, options ...OptionFunc) (valid bool, err error) {
+	if content == "" {
+		return false, errors.New("content is missing")
+	}
+	opts := struct {
+		Content string `json:"content"`
+	}{
+		Content: content,
+	}
```

Comment:
The convention in this package is to define a named type for this, instead of an anonymous inline struct.